app/access_code_cast_strings.grdp

```diff
@@ -2,31 +2,40 @@
 <!-- Access Code Cast-specific strings (included from generated_resources.grd). -->
 <!--This file is created by l10nUtil.js. Do not edit manually.-->
 <grit-part>
+  <message name="IDS_ACCESS_CODE_CAST_ACCESS_CODE_MESSAGE" desc="Text shown in the access code cast dialog that describes that the access code is found on a Bravecast device">
```

Comment:
Overly aggressive substitution here, needs to be rolled back to Chromecast.

Reply:
Thank you, fixed in 19cc0c8f436703ccbd500f16944a4c2089ef2d27.